src/Symfony/Component/HttpClient/Retry/RetryDeciderInterface.php

```diff
@@ -23,5 +25,5 @@ interface RetryDeciderInterface
      *
      * @return ?bool Returns null to signal that the body is required to take a decision
      */
-    public function shouldRetry(string $requestMethod, string $requestUrl, array $requestOptions, int $responseStatusCode, array $responseHeaders, ?string $responseContent): ?bool;
+    public function shouldRetry(int $retryCount, string $requestMethod, string $requestUrl, array $requestOptions, int $responseStatusCode, array $responseHeaders, ?string $responseContent, ?TransportExceptionInterface $exception): ?bool;
```

Comment:
Please add docs for `$retryCount`. If I make request and it fails. I ask this RetryDecider for the first time. Will `$retryCount` be 0 or 1? Please add that in the doc block.

Reply:
ahahah.. I had to re-read code to write the documentation. You're absolutely right on this!